src/reactor.cc

```diff
@@ -158,10 +175,25 @@ std::set<cyclus::RequestPortfolio<Material>::Ptr> Reactor::GetMatlRequests() {
   std::set<RequestPortfolio<Material>::Ptr> ports;
   Material::Ptr m;
 
-  int n_assem_order =
-      n_assem_core - core.count() + n_assem_fresh - fresh.count();
+  // second min expression reduces assembles to amount needed until
+  // retirement if it is near.
+  int n_assem_order = n_assem_core - core.count() + n_assem_fresh - fresh.count();
```

Comment:
the variable names used in this section are inconsistent, e.g., `n_assem_order` vs. `ncyclesleft`. The GCSG says either is fine. https://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Variable_Names 

Perhaps we can adopt a convention since we don't require trailing underscores in member variables. I'll let you make a decision.